app/src/main/java/de/danoeh/antennapod/activity/MediaplayerActivity.java

```diff
@@ -504,7 +505,7 @@ public void onTimerSet(long millis, boolean shakeToReset, boolean vibrate) {
     private static String getWebsiteLinkWithFallback(Playable media) {
         if (media == null) {
             return null;
-        } else if (media.getWebsiteLink() != null) {
+        } else if (StringUtils.isNotBlank(media.getWebsiteLink())) {
```

Comment:
If you use Android's `TextUtils.isEmpty` instead, we do not need to use apache commons here. I think using the Android methods should be preferred.

Reply:
1. `isBlank` catches more cases (e.g., "   ") . 
2. If we use TextUtils here, do we use it here in `FeedItemUtils` too? But if we use TextUtils there, then the method can't be unit-tesed.
3. the apache commons text are used in dependent libraries so it's not really a new depedency, based on what dexCount reported.

Reply:
Okay, we can keep it :)

Reply:
I wonder if other projects have went through similar conversation and ended up using apache commons.